view/frontend/web/js/view/payment/alma_monthly_payments.js

```diff
@@ -28,26 +28,31 @@ define(
     [
         'uiComponent',
         'Magento_Checkout/js/model/payment/renderer-list',
-        'Magento_Customer/js/customer-data'
+        'Magento_Customer/js/customer-data',
+        'Magento_Checkout/js/model/quote',
     ],
     function (
         Component,
         rendererList,
-        customerData
+        customerData,
+        quote
     ) {
         'use strict';
+        quote.totals.subscribe(function(){
+            reloadAlmaSection();
+        }, null, 'change');
 
-        var methodCode = 'alma_monthly_payments';
-        var almaSectionName = 'alma_section';
+        function reloadAlmaSection() {
+            customerData.reload(['alma_section'])
+        }
 
-        if (customerData.get(almaSectionName)().allPaymentPlans.length > 0) {
+        var methodCode = 'alma_monthly_payments';
```

Comment:
You probably have to re-indent code bellow

Reply:
done